tests/test_modeling_reformer.py

```diff
@@ -571,7 +597,9 @@ def test_model_from_pretrained(self):
 @require_torch
 class ReformerLSHAttnModelTest(ReformerTesterMixin, ModelTesterMixin, unittest.TestCase):
     all_model_classes = (
-        (ReformerModel, ReformerModelWithLMHead, ReformerForQuestionAnswering) if is_torch_available() else ()
+        (ReformerModel, ReformerModelWithLMHead, ReformerForSequenceClassification, ReformerForQuestionAnswering)
+        if is_torch_available()
+        else ()
```

Comment:
test_pruning = False
    test_headmasking = False
    test_torchscript = False


Obviously out of scope, but you could try turning some of these on and seeing whether/why they break.